packages/logging/src/index.ts

```diff
@@ -0,0 +1,151 @@
+import { Version } from './shared';
+export interface Transport {
```

Comment:
This seems to be a duplicate, I see it defined under `./transports/index.ts`. I would move it to its own file under `src/transport.ts`